portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>Hobbies</h1>
+
+    <h2>Sports</h2>
+
+    <div class = "hobby">
+      <h4>Tennis</h4>
+      <p>
+          Tennis is easily defined as my favorite sport. I started playing with my father when I
+          was around four years old. He would take my brother and I out to play even though we were
+          not able to properly hit the ball over the net yet. Over the years, what used to be an 
+          activity meant to bring us together as a family transitioned into an activity I am
+          deeply passionate about.
+      </p>
+      <p>
+          I joined my first team and got my first coach as a freshman in high school. I joined
+          varsity right away and played at that level for all four years. Throughout my high school
+          career I earned many league awards and finished first or second in the league for three
+          consecutive years.
+      </p>
+      <p>
+          I still love tennis to this day, and I will continue practicing to participate in local 
+          tournaments.
+      </p>
+    </div>
+
+    <div class = "hobby">
+	  <h4>Soccer</h4>
+	  <p>
+		  Before I fell in love with tennis, I enjoyed soccer. Prior to entering high school, I had no
+		  friends that played tennis. So, I directed my energy towards soccer.
+	  </p>
+      <p>
+	      I played for many years, with a brief period of inactivity due to health concerns, but I was
+	      never that good. I gradually started to enjoy soccer less and less, but I will always watch
+	      the FIFA World Cup. During the last tournament, Peru had finally qualified after 36 years of
+	      not making an appearance in the tournament. Vamos Perú!
+      </p>
+    </div>
```

Comment:
Be careful mixing tabs and spaces

```suggestion
    <div class = "hobby">
      <h4>Soccer</h4>
      <p>
        Before I fell in love with tennis, I enjoyed soccer. Prior to entering high school, I had no
        friends that played tennis. So, I directed my energy towards soccer.
      </p>
      <p>
        I played for many years, with a brief period of inactivity due to health concerns, but I was
        never that good. I gradually started to enjoy soccer less and less, but I will always watch
        the FIFA World Cup. During the last tournament, Peru had finally qualified after 36 years of
        not making an appearance in the tournament. Vamos Perú!
      </p>
    </div>
```